app/src/main/java/co/tinode/tindroid/media/SpanFormatter.java

```diff
@@ -271,7 +317,8 @@ public void onClick(View widget) {
                         }, content);
                     } catch (ClassCastException | NullPointerException ignored) {}
                     break;
-                case "MN": break;
+                case "MN":
+                    break;
```

Comment:
This seems to be an unnecessary change. If you want to just reformat the code then please do it in a separate PR.